TestProjects/ShaderFoundry/Assets/CommonAssets/Editor/Targets/SRP/SimpleSRPFoundryPipeline.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace UnityEditor.Rendering.Foundry
+{
+    public class SimpleSRPFoundryPipelineInstance : RenderPipeline
+    {
+        SimpleSRPFoundryPipeline m_Parent;
+
+        public SimpleSRPFoundryPipelineInstance(SimpleSRPFoundryPipeline parent)
+        {
+            m_Parent = parent;
+        }
+
+        protected override void Render(ScriptableRenderContext renderContext, Camera[] cameras)
+        {
+            m_Parent.Render(renderContext, cameras);
+        }
+    }
+
+    [ExecuteInEditMode]
+    public class SimpleSRPFoundryPipeline : RenderPipelineAsset
+    {
+#if UNITY_EDITOR
+        [UnityEditor.MenuItem("RenderPipeline/Create SimpleSRPFoundryPipeline")]
```

Comment:
I think the `#if UNITY_EDITOR` is fine to have, but is there any chance that any of this code actually gets run in a player (i.e., do we actually _need_ it)?

Reply:
This was copied from the SRP Batcher tests custom SRP.
This creates an menu option to make it easy to create a new asset file for the srp pipeline. The pipeline is then used as is, but the code to create a new asset isn't needed outside of the editor.

Reply:
This is under "<>/Editor/<>" folder in the structure already, so I think it won't get included in the player anyway